command/command.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import "github.com/kanisterio/safecli"
+
+// New creates a new safecli.Builder with the given name and arguments.
+func New(name string, args ...Applier) (*safecli.Builder, error) {
+	cmd := safecli.NewBuilder(name)
+	if err := Apply(cmd, args...); err != nil {
```

Comment:
Since `command.New` is the way this lib is going to be used, what's the purpose of `Apply` function? Is there an example of usage of `Apply` outside of this?

Reply:
Good catch, let's make this function private.